jwst/cube_skymatch/skymatch.py

```diff
@@ -74,12 +66,6 @@ def match(skycubes, subtract=False):
     # Time it
     runtime_begin = datetime.now()
 
-    log.info(" ")
-    log.info("***** {:s}.{:s}() started on {}"
-             .format(__name__, function_name, runtime_begin))
-    log.info("      Version {} ({})".format(__version__, __vdate__))
-    log.info(" ")
```

Comment:
This kind of logging info is done automatically for each step from the base class `Step`. If more logging is needed it should not use `__vdate__ ` and `__version__`.